src/Resources/Locales/en.json

```diff
@@ -1392,11 +1395,20 @@
                 "messages": {
                     "noColor": "Not set",
                     "loading": "Loading historized data...",
-                    "noData": "No historized data found in selected time frame.",
+                    "noData": "No historized data found in the selected time range.",
                     "noSeries": "No twin selected to query, please add twin properties to view historized data."
                 }
             },
             "noSeriesDescription": "To view data history for this twin, select a property to view."
         }
+    },
+    "dataHistoryErrorHandlingWrapper": {
+        "errors": {
+            "genericNetwork": "Cluster cannot be connected which may be due to invalid url, stopped service or a network error.",
+            "genericBadRequest": "Bad request exception occured.",
```

Comment:
do we have to include the bad request part? THat's very jargony. Can we be generic and say `Something went wrong, please try again` or `An exception occurred with the request`. I personally prefer #1.

Reply:
Since I got the errors from [here](https://learn.microsoft.com/en-us/azure/data-explorer/kusto/api/netfx/kusto-data-client-errors#bad-request-exceptions), I wanted to reflect the error messages as is but I can change it to that one